### PR TITLE
layout cut only appears on index and location page

### PR DESCRIPTION
**Pull request by naps62**

### CODE REVIEW

File: html.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { prefixLink } from 'gatsby-helpers';
 
 const BUILD_TIME = new Date().getTime();
 
-const Html = ({ body }) => {
+const Html = ({ body } = props) => {
```
Comment (pfac) [inline html.js]: It looks to me like `props` is not being used. Why did you have to add it?

File: pages/_template.jsx [hunk 2]
```diff
@@ -21,6 +21,7 @@ export default class Template extends Component {
 
   constructor(props) {
     super(props);
+    console.log(props)
```
Comment (pfac) [inline pages/_template.jsx]: :warning:

File: pages/_template.jsx [hunk 3]
```diff
@@ -32,6 +33,15 @@ export default class Template extends Component {
   toggleOverlayMenu = () =>
     this.setState({ overlayOpen: !this.state.overlayOpen });
 
+  renderCut() {
+    console.log(this.props.location.pathname)
```
Comment (pfac) [inline pages/_template.jsx]: :warning:

**Review by pfac — COMMENTED** [review]
:question:

File: html.js (hunk 1 above)
Comment (naps62) [reply html.js]: it looks like a left a lot of debug stuff here. shame on he

**Comment by naps62** [comment]
@pfac all fixed

**Review by pfac — APPROVED** [review]
:+1:

**Pull request merged**